packages/core/scripts/gas-rebate/rebates/Rebate_1.json

```diff
@@ -1,152 +1,223 @@
-    "0x1b52a70616ee7ed4ea070d19fd18a60e467a5276": 10.193636428435054,
-    "0xe7a6d81a4b543c1bf2bd10ceb74a72689dc2e4b1": 2.830057183661909,
-    "0xd2a78bb82389d30075144d17e782964918999f7f": 11.745860458482241,
-    "0x9ae57ce31bdae73a0d3bf701998af5494a1ceeec": 3.47488721389034,
-    "0x230d31eec85f4063a405b0f95bde509c0d0a8b5d": 2.8546355058937887,
-    "0xdb2808ed6cf0b8b9686222689cf4f6e88e42b80a": 11.502520041384647,
-    "0xbe3bf9bb42bf423695b792250be206c9a58e2de1": 3.525179757583272,
-    "0xf288464959a5dd1671bce3458a3a4ed0cc314e3c": 3.4744802700582764,
-    "0x6888ff4be3262f3232e1c022de731680a16ddda7": 12.047012758560587,
-    "0x605acc13c07cb2de5261dc64d315857fde7d5c5c": 10.673980723280241,
-    "0xf627e5f4bad95a956468d8bb6ee20b119f992e96": 11.40455460930791,
-    "0xde3af8bb46d556df3d7c99cc313c4b5713b4a5a5": 11.097924423846756,
-    "0xbf87c98f8a9c1f2a88b613813ec1e974105eb615": 11.370145860230917,
-    "0x14a29ccfe3c1cc777a6f3a2ead6f07cec5cc5dc1": 3.8113307227465403,
-    "0x67d24f48f1b58c4d4dcda57ba2386dce76353fa2": 12.051552262185249,
-    "0xb2a5e135892b92d8554dab9adc5291ee65a9957b": 2.7986214307381942,
-    "0x4c39185a078b5666c372538231cb793a0928807b": 10.930939281158391,
-    "0x120219511b9b0254e7d15014db83b8752289fda3": 10.691221617522928,
-    "0xccd9c5465fac5c0df86fd9b7d524a5949defcef6": 4.030475242557726,
-    "0xfc119030940852a33fbb787a6fd086efcabe458f": 10.406832306558663,
-    "0xf7b10d603907658f690da534e9b7dbc4dab3e2d6": 11.628170141241998,
-    "0x7553fa99ab1f429551eec660708c08e14e30584f": 8.314068305079523,
-    "0x8d680d96befc53d8b90f76d1e8e99e842b3387a2": 9.132647868581346,
-    "0xbee33ac5520a2245cbf8288e768a5cc26927cddf": 9.770010522871106,
-    "0xef22364cdabd7f39a3b6fd836f059336c810a8a9": 2.4720088223633803,
-    "0x1757d400578cf8a2f3da8228e4402fb1bb3fc968": 2.0110858874753648,
-    "0x56d14fcb4587b5845fa4823af790672ebf92580d": 2.028511745130638,
-    "0xf307077dd0e27a382e93f2e3d13a9c6584582332": 7.579814685325461,
-    "0x8d07d225a769b7af3a923481e1fdf49180e6a265": 1.978616638119834,
-    "0x099686478448ce02f8e61c6b01904d0548daeae5": 7.433732167838405,
-    "0xa66b7e13467084f248e7d731eb2f9ff8e43db1cf": 1.6222811342836554,
-    "0x966cf5cd0624f1efcf21b0abc231a5ccc802b861": 8.418561988338048,
-    "0x57c458c1354fee3aba912794709c91180869f953": 1.6469951044595568,
-    "0xedc63f732d61df2556fd3ae84f911a84ac64b4ba": 1.622129978502763,
-    "0xe311958ac79b41bf3a0595470a964b8efbe0fb36": 9.477010142669442,
-    "0x28d804bf2212e220bc2b7b6252993db8286df07f": 1.680771648217519,
-    "0xe6662ec7fc7c84231b29f0f0da6321673d100854": 1.6218741904515916,
-    "0x8bd16de8938a2ad16794ac7e7502896f156e370c": 8.408517356682127,
-    "0x2f840532892f91811f99cdaec703f322a779cc66": 8.039390044761545,
-    "0x83b19521180d533dbae360be77f95d0da3aef98e": 11.086509471110434,
-    "0xa08aca4a5d1a6813dcae969df892581047cfcc30": 6.674638805759128
-  }
-}
+    "rebate": 1,
+    "fromBlock": 10752294,
+    "toBlock": 10869637,
```

Comment:
this block changed, is that ok? also was this meant to regen this file?

Reply:
yes this is fine, this Rebate_1.json file is unofficial